web-animations-2/Overview.bs

```diff
@@ -388,15 +390,38 @@ After the step to assign <var>new effect</var> as <var>animation</var>'s
 
 <h4 id='waiting-for-the-associated-effect'>Waiting for the associated effect</h4>
 
-The definition of when an animation is ready needs to be extended to consider
-descendant effects <ins cite="#amend-CE">and custom effects</ins> such that the first condition is:
+Replace:
 
+>  An animation is ready at the first moment
+>  where <em>both</em> of the following conditions are true:
+>
+>  * the user agent has completed any setup required
+>      to begin the playback of the animation's [=associated effect=],
+>      including rendering the first frame of any [=keyframe effect=].
+>
+>  * the animation is associated with a [=timeline=]
+>      that is not [=inactive timeline|inactive=].
+
+With:
+
+>  An animation is <dfn>ready</dfn> at the first moment
+>  where <em>all</em> of the following conditions are true:
+>
 > *   the user agent has completed any setup required to begin the playback of
 >     each <a>inclusive descendant</a> of
 >     the animation's <a>associated effect</a>
 >     including rendering the first frame of any <a>keyframe
 >     effect</a><ins cite="#amend-CE"> or executing any <a>custom effects</a> associated with an
 >     <a>animation effect</a></ins>.
+>
+> *   the animation is associated with a [=timeline=]
+>     that is not [=inactive timeline|inactive=].
+>
+> *   the animation's [=hold time=] or [=animation/start time=] is resolved.
```

Comment:
Arguably knowing the hold time and start time is one of the requirements for the first point - i.e. we can't "render" the first frame until we know this. But I suppose it doesn't hurt to call out explicitly :-).

Reply:
ack